test/pom.xml

```diff
@@ -39,13 +39,13 @@
         <dependency>
             <groupId>io.debezium</groupId>
             <artifactId>debezium-core</artifactId>
-            <version>0.8.1.Final</version>
+            <version>${debezium.version}</version>
```

Comment:
Just in case you didn't know: If you add a `<dependency>` to the `<dependencyManagement>` in the top-level pom.xml it means you don't need a `<version>` when you mention the `<dependency>` in a `<depencencies>` section of a child pom.

Reply:
I do now. But I wasn't sure whether doing it the way it was had some reason or not. So I decided not to touch it. Do you want me to change it?

Reply:
Ideally, yes please.

Reply:
Ok, should be done now.